Microsoft.Bot.Builder.sln

```diff
@@ -814,8 +814,8 @@ Global
 		{19CE18FD-3F99-4994-B88E-7D32F9303547}.Debug-Windows|Any CPU.Build.0 = Debug|x64
 		{19CE18FD-3F99-4994-B88E-7D32F9303547}.Release|Any CPU.ActiveCfg = Release|x64
 		{19CE18FD-3F99-4994-B88E-7D32F9303547}.Release|Any CPU.Build.0 = Release|x64
-		{19CE18FD-3F99-4994-B88E-7D32F9303547}.Release-Windows|Any CPU.ActiveCfg = Release|Any CPU
-		{19CE18FD-3F99-4994-B88E-7D32F9303547}.Release-Windows|Any CPU.Build.0 = Release|Any CPU
+		{19CE18FD-3F99-4994-B88E-7D32F9303547}.Release-Windows|Any CPU.ActiveCfg = Release|x64

```

Comment:
This looks unintentional.  Please switch this back to 'Any CPU'

Thanks for this PR!

Reply:
Thanks for pointing this out. I believe that happened when I built the solution in Visual Studio.